cli/commands/configure/configure.go

```diff
@@ -115,11 +115,11 @@ func Command(cli *cli.SensuCli) *cobra.Command {
 	}
 
 	_ = cmd.Flags().BoolP("non-interactive", "n", false, "do not administer interactive questionnaire")
-	_ = cmd.Flags().StringP("url", "", cli.Config.APIUrl(), "the sensu backend url")
```

Comment:
What if a user runs `sensuctl configure` after they have initially configured? If they leave these flags blank, wouldn't their current configuration be overwritten by the defaults?